### PR TITLE
Second chunk of terrain descriptions

### PR DESCRIPTION
**Pull request by DracoGriffin**
Another round of descriptions to be reviewed.

Some descriptions are a bit obtuse since they are constructed by player and the construction menu goes by roofs, not floors -- additionally as floors and roofs are treated as the same terrain by code.

Specific vocabulary is used for players to more readily identify how terrain may be interacted with (example would be examine, as some terrains can be interacted through examine hotkey).

Furthermore, descriptions may not always be 100% appropriate as different locations reference terrain in different ways (pavements, sidewalks and such -- especially "walkway" and "metal grate" aren't referenced properly in maps so did my best to describe more broadly).

### CODE REVIEW

**Comment by Leland** [comment]
Yeah I'll try to get some notes in on this today

File: data/json/terrain.json [hunk 1]
```diff
@@ -257,6 +269,7 @@
         "type" : "terrain",
         "id" : "t_ov_reb_cage",
         "name": "rebar roof cage",
+        "description": "A network of architecturally sound rebar in order to support a floor and roof, looks like it's missing the poured concrete.",
```
Comment (Leland) [inline data/json/terrain.json]: > architecturally sound rebar in order to support a floor and roof, looks like it's missing the poured concrete

Everything after "rebar" is a separate clause. Could read:

> A network of architecturally sound rebar. Could be used to support a floor and roof, though it's missing the poured concrete.

File: data/json/terrain.json [hunk 2]
```diff
@@ -278,6 +291,7 @@
         "type" : "terrain",
         "id" : "t_strconc_floor_halfway",
         "name": "half-built reinforced concrete floor",
+        "description": "Unfinished series of rebar and poured concrete, the floor hasn't been smoothed and the roof isn't quite filled in yet.",
```
Comment (Leland) [inline data/json/terrain.json]: The comma is here being used to separate two distinct clauses, should be a period (preferably), a semicolon or a hyphen.

> Unfinished series of rebar and poured concrete. The floor hasn't been smoothed and the roof isn't quite filled in yet.

File: data/json/terrain.json [hunk 3]
```diff
@@ -300,6 +314,7 @@
         "type" : "terrain",
         "id" : "t_dirt",
         "name": "dirt",
+        "description": "It's dirt. Looks like some fine soil for tillage, or dug out for construction projects.",
```
Comment (Leland) [inline data/json/terrain.json]: As mentioned in the previous PR, the second sentence reads a bit funny.

> Looks like some fine soil for tillage, or dug out for construction projects.

First, "tillage" should be "tilling," the verb. "Tillage" is the method/system of tilling.  Secondly, the two clauses rely on the same predicate ("looks like some fine soil"), and it's causing some confusion. The second clause is read as:

> Looks like some fine soil for dug out for construction projects.

Here are some alternatives:

> It's dirt. The fine soil could be tilled for crops or dug out for construction projects.

> It's dirt. Looks like some fine soil for tilling. Could also be dug out for construction projects.

> It's dirt, ideal for tilling. Could also be dug out for construction projects.

File: data/json/terrain.json [hunk 4]
```diff
@@ -348,6 +365,7 @@
         "type" : "terrain",
         "id" : "t_dirtmound",
         "name": "mound of dirt",
+        "description": "An area of heaped dirt, not easily traversible but if examined more closely, they could be quite favorable for planting seeds and the like.",
```
Comment (Leland) [inline data/json/terrain.json]: Should be "traversable"

Additionally, "not easily traversible" and "could be quite favorable for planting seeds" are not at odds with one another – no need for a "but."

Finally, "planting seeds and the like." Do we have things other than seeds that could be planted? 

Perhaps:

> An area of heaped dirt, not easily traversed. Could be quite favorable for planting seeds.

File: data/json/terrain.json [hunk 5]
```diff
@@ -540,6 +568,7 @@
         "type" : "terrain",
         "id" : "t_grass",
         "name": "grass",
+        "description": "A beautiful, if unkempt section of hardy Kentucky blue grass.  Cutting the grass short enough would destroy the root system, causing this area to turn into a patch of dirt.",
```
Comment (Leland) [inline data/json/terrain.json]: "if unkempt" is a parenthetical statement, and should have a closing comma as well. I would recommend using either parentheses or hyphens to more abruptly separate it, however.

> A beautiful - if unkempt - section of hardy Kentucky...

Also: "bluegrass" is one word

File: data/json/terrain.json [hunk 6]
```diff
@@ -556,6 +585,7 @@
         "type" : "terrain",
         "id" : "t_grass_white",
         "name": "white grass",
+        "description": "A section of Kentucky blue grass covered in white latex-based paint, with neat lines designed for recreational sports.",
```
Comment (Leland) [inline data/json/terrain.json]: "bluegrass"

File: data/json/terrain.json [hunk 7]
```diff
@@ -587,6 +618,7 @@
         "type" : "terrain",
         "id" : "t_pavement",
         "name": "pavement",
+        "description": "A segment of asphalt, slowly degrading from cracks, frost heaves and lack of maintenance.",
```
Comment (Leland) [inline data/json/terrain.json]: CDDA uses the Oxford comma

File: data/json/terrain.json [hunk 8]
```diff
@@ -643,6 +677,7 @@
         "type" : "terrain",
         "id" : "t_concrete",
         "name": "concrete",
+        "description": "A newer segment of poured concrete with surface finishes for aesthetics and resistant to freeze-thaw cycles.",
```
Comment (Leland) [inline data/json/terrain.json]: Should be "resistance," the adjective

File: data/json/terrain.json [hunk 9]
```diff
@@ -795,6 +838,7 @@
         "type" : "terrain",
         "id" : "t_wall_log_half",
         "name": "half-built log wall",
+        "description": "A half-constructed wall of notched logs that interlock to provide stability.  Needs a few more logs to hold up a roof, and looks flammable.",
```
Comment (Leland) [inline data/json/terrain.json]: > Needs a few more logs to hold up a roof, and looks flammable

These clauses are not related to one another. Comma should probably be a period

File: data/json/terrain.json [hunk 10]
```diff
@@ -868,6 +915,7 @@
         "type" : "terrain",
         "id" : "t_palisade",
         "name": "palisade wall",
+        "description": "An age old type of fortification consisting of hefty lumber, staked into the ground and cabled together.",
```
Comment (Leland) [inline data/json/terrain.json]: Should be "age-old"

Also, no need for a comma here ("staked into the ground" is not the second item in the list – it directly relates to the subject, "hefty lumber")

File: data/json/terrain.json [hunk 11]
```diff
@@ -887,6 +935,7 @@
         "type" : "terrain",
         "id" : "t_palisade_gate",
         "name": "palisade gate",
+        "description": "A large doorway consisting of long logs cabled together, that could be widened.  Can act as a door if some type of pulley system were rigged on an adjacent palisade wall.",
```
Comment (Leland) [inline data/json/terrain.json]: What does "that could be widened" mean?

File: data/json/terrain.json [hunk 12]
```diff
@@ -984,6 +1037,7 @@
         "type" : "terrain",
         "id" : "t_wall_wood_broken",
         "name": "broken wood wall",
+        "description": "A number of planks are missing, and the structure is beginning to sag and fall apart.  It's going to need quite a bit of work to repair this wall.",
```
Comment (Leland) [inline data/json/terrain.json]: Just a note to you as an author:

Don't be afraid to leave out commas! English allows one to be _very_ judicial with comma use (just ask [The Constitution](http://www.nytimes.com/2007/12/16/opinion/16freedman.html)) but sentences generally read more forcefully and easily with as few as one needs. It's kind of the Hemingway approach.

> A number of planks are missing, and the structure is beginning to sag and fall apart

This is grammatically fine. However, the comma introduces a pause, and one that's perhaps not necessary. Here and elsewhere consider:

> A number of planks are missing and the structure is beginning to sag and fall apart

File: data/json/terrain.json [hunk 13]
```diff
@@ -1051,6 +1107,7 @@
             "t_wall_metal_v"
         ],
         "name": "metal wall",
+        "description": "An industrially fabricated thick sheet of carefully positioned and joined seamlessly with perimeter sealant, this wall is capable of resisting extreme elements as well as hostile forces.  Blast load rated and extremely fire-retardant, any breaches will require specialized tools or industrial vehicles.",
```
Comment (Leland) [inline data/json/terrain.json]: > ...[a] thick sheet of carefully positioned and joined seamlessly with perimeter sealant

What's carefully positioned?

> any breaches will require specialized tools or industrial vehicles

Consider "breaching will require," since it's the act of breaching that will require those items

**Comment by illi-kun** [comment]
Just a friendly notice: not all of Leland's comments were addressed \ implemented.

**Comment by DracoGriffin** [comment]
> Just a friendly notice: not all of Leland's comments were addressed \ implemented.

Half of them are from the first chunk of descriptions, which was already merged but because I guess I created this PR so quickly after the merging of the first, it doesn't seem to account for those, so those were already addressed/responded to in #23046 (which @Leland reviewed the #23036 which was much too long)

**Comment by illi-kun** [comment]
Merge conflicts should be solved before merge anyway.

File: data/json/terrain.json (hunk 3 above)
Comment (DracoGriffin) [reply data/json/terrain.json]: Correct, tillage is the noun for 
> the preparation of land for growing crops.

So to illustrate its usefulness here: "Looks like some fine soil for hydroponics" - it wouldn't be "Looks like some fine soil for hydroponically". So to correct for "tilling", it would be stated as "Tilling this fine soil would be useful ... ". So, "tillage" is the correct form here.

The second part could be clunky, depending on the reader.

File: data/json/terrain.json (hunk 4 above)
Comment (DracoGriffin) [reply data/json/terrain.json]: Yeah, not sure how that was missed. Google seems to show both, but traversible is the depreciated form.

Wild vegetable stems, rhubarb stems, hop rhizomes, garlic cloves, raw beans, seed potato (although does have seed in the name), mushroom spores and morel mushroom spores. These are for "the like". And also allows for later changes if re-planting trees ever becomes a thing.

File: data/json/terrain.json (hunk 5 above)
Comment (DracoGriffin) [reply data/json/terrain.json]: Looks like I must've copied and pasted an older edit, as I could've sworn I changed these, as mentioned in the other comment.

**Comment by DracoGriffin** [comment]
Conflicts are from changed lines, so unable to fix those.

Addressed Leland's comments.

Should be ready now.

**Comment by illi-kun** [comment]
1) Please don't put [RDY] tag to PR's name, it's useless. Each PR is ready by default if tag [WIP] is not applied.
2) This branch has conflicts that must be resolved. Do you see a button 'Resolve conflicts'? You can use it to fix the issue. Maybe [this article](https://help.github.com/articles/resolving-a-merge-conflict-on-github/)  can be useful for you.

**Comment by DracoGriffin** [comment]
> Please don't put [RDY] tag to PR's name, it's useless. Each PR is ready by default if tag [WIP] is not applied.

Just following examples of other PRs.

> This branch has conflicts that must be resolved. Do you see a button 'Resolve conflicts'? You can use it to fix the issue. Maybe this article can be useful for you.

Yes, this was useful, I didn't realize they could be fixed that way. Still learning GitHub if it isn't evident by my lack of coding ability.

**Comment by illi-kun** [comment]
> Just following examples of other PRs.

Yeah, i've cleaned them up from other PRs too. As a Russian proverb says: 'bad habits are the most contagious'.

> Yes, this was useful

It's nice to hear that, thanks for quick respond.
